Shared/DemoParameters.cs

```diff
@@ -9,7 +9,7 @@ namespace LaunchDarkly.Hello
     public class DemoParameters
     {
         // Set MobileKey to your LaunchDarkly mobile key.
-        public const string MobileKey = "mob-f0c2ce9a-fdca-4190-b0da-8f6645a8330e";
+        public const string MobileKey = "";
```

Comment:
At some point in the past, we were pre-populating this with a key for an environment in the sandbox account so that the demo could be run totally as-is. But our new standard is for the hello demos not to have any default key; we add one from an environment variable in CI, so that we can verify that the demo works, but a user checking out the demo has to add in their own key.